src/prisma/schema.prisma

```diff
@@ -11,7 +11,25 @@ generator client {
 }
 
 model User {
-  id       Int    @id @default(autoincrement())
-  email    String @unique
+  id       Int      @id @default(autoincrement())
+  email    String   @unique
```

Comment:
this changes are not needed and don't make any changes to the code functionality, try to avoid pushing code that will be not readable because you are not doing code reviews better, we can discuss this changes and make it on a refactor branch